pyiron_atomistics/atomistics/master/phonopy.py

```diff
@@ -18,7 +18,8 @@
 from pyiron_atomistics.atomistics.structure.phonopy import (
     publication as phonopy_publication,
 )
-from pyiron_base import state, JobGenerator, deprecate
+from pyiron_base import state, JobGenerator
+from pyiron_snippets.deprecate import deprecate
 from pyiron_snippets.import_alarm import ImportAlarm
```

Comment:
Especially this line would be much cleaner as: 
```suggestion
from pyiron_snippets import deprecate, ImportAlarm
```

Reply:
As mentioned above, I don't want this because if snippets grows I don't want to burn time importing everything in it just because you use one thing in it. But tbh I'm not sure that even with a large module this burns significant time, so if you have data to prove it won't impact performance down the line, then I'm open to it.

Reply:
The current way keeps the different submodules of snippets and their API uncoupled as is intended and so I think it is strongly preferable to importing/exporting everything in the top module.  Especially since it would be for purely aesthetic reasons.  `from pyiron_snippets.deprecate import *` would work just as well.